packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -452,23 +450,26 @@ export function buildRootObject(powers, _params, testJigSetter = undefined) {
 
     // First, evaluate the contract code bundle.
     const contractCode = evalContractBundle(bundle);
-    contractCode.catch(err => {
-      // Remove to suppress Node.js's UnhandledPromiseRejectionWarning
-      throw err;
-    });
+    // Don't trigger Node.js's UnhandledPromiseRejectionWarning.
+    // This does not suppress any error messages.
+    contractCode.catch(() => {});
 
     // Next, execute the contract code, passing in zcf
-    const { creatorFacet, publicFacet, creatorInvitation } = await E(
-      contractCode,
-    ).start(zcf);
-
     /** @type {Promise<ExecuteContractResult>} */
-    return harden({
-      creatorFacet,
-      publicFacet,
-      creatorInvitation,
-      addSeatObj,
-    });
+    const result = E(contractCode)
```

Comment:
This example makes clear the cost of #2030 . Once that is fixed, this can just be

```suggestion
    return E(contractCode)
```

and avoid even creating the separate `result` variable.